doc/conf.py

```diff
@@ -264,6 +265,9 @@
 # If true, do not generate a @detailmenu in the "Top" node's menu.
 #texinfo_no_detailmenu = False
 
+# Mapping for external documentation in Sphinx.
+intersphinx_mapping = {'orange3': ('../../orange3/doc/visual-programming/build/html',
+                                   '../../orange3/doc/visual-programming/build/html/objects.inv')}
```

Comment:
I don't have a clue. 🙈 This was actually just a test and probably shouldn't be in the PR as it doesn't work at all. Thanks, Sphinx, truly. 😠